rd-net/Lifetimes/Lifetimes.csproj

```diff
@@ -18,6 +18,10 @@
         
     </PropertyGroup>
     
+    <PropertyGroup Condition="'$(TargetFramework)' == 'net35'">
+        <WarningsNotAsErrors>CS8632</WarningsNotAsErrors>
+    </PropertyGroup>
+    
```

Comment:
Should this actually go to the `Directory.Build.props`?

Actually wait a minute, our `Directory.Build.props` already has `<NoWarn>nullable</NoWarn>` for this TFM. Doesn't it work?

Reply:
Probably it doesn't work for Rider.  Rider shows a lot of errors in SWEA.

Reply:
Let's unvestigate; I was unable to reproduce.